qt/python/mantidqt/widgets/memorywidget/memorypresenter.py

```diff
@@ -25,18 +25,33 @@ class MemoryPresenter(object):
     """
     def __init__(self, view):
         self.view = view
-        self.timer = QTimer()
+        self.timer = Timer(TIME_INTERVAL_MEMORY_USAGE_UPDATE, self.update_memory_usage)
+        self.thread_on = False
+        self.updating_cancelled = False
+        self.application_closing = False
         self.update_memory_usage()
         self.set_bar_color_at_start()
-        self.update_at_regular_intervals()
+        self.start_memory_widget_update_thread()
```

Comment:
As we are inside the presenter of the widget perhaps we shorten this to `self.start_update_thread`?